pkg/reconciler/common/transformers.go

```diff
@@ -587,3 +588,26 @@ func AddConfiguration(config v1alpha1.Config) mf.Transformer {
 		return nil
 	}
 }
+
+func CreateOperatorVersionConfigMap(manifest mf.Manifest, obj v1alpha1.TektonComponent) error {
+	koDataDir := os.Getenv(KoEnvKey)
+	operatorDir := filepath.Join(koDataDir, "info")
+
+	if err := AppendManifest(&manifest, operatorDir); err != nil {
+		return err
+	}
+
+	manifest, err := manifest.Transform(
+		mf.InjectNamespace(obj.GetSpec().GetTargetNamespace()),
+		mf.InjectOwner(obj),
+	)
+	if err != nil {
+		return err
+	}
+
+	if err = manifest.Apply(); err != nil {
+		return err
+	}
+
+	return nil
+}
```

Comment:
CreateOperatorVersionConfigMap is not of type `transformer`(`type Transformer func(u *unstructured.Unstructured) error`. 

So could you move this function `CreateOperatorVersionConfigMap`and `createTargetNamespace` to a file called `targetnamespace.go`in common.